forest/cli/src/cli/info_cmd.rs

```diff
@@ -0,0 +1,374 @@
+            "calibnet".to_string(),
+            None,
+            None,
+        )
+    }
+
+    #[quickcheck]
+    fn test_sync_status_ok(duration: Duration) {
+        let tipset = mock_tipset_at(duration.as_secs() + (EPOCH_DURATION_SECONDS as u64 * 3 / 2));
+
+        let status = node_status(duration, tipset.as_ref());
+
+        assert_ne!(status.sync_status, SyncStatus::Slow);
+        assert_ne!(status.sync_status, SyncStatus::Behind);
+    }
+
+    #[quickcheck]
+    fn test_sync_status_behind(duration: Duration) {
+        let duration = duration + Duration::from_secs(300);
+        let tipset = mock_tipset_at(duration.as_secs().saturating_sub(200));
+        let status = node_status(duration, tipset.as_ref());
+
+        assert!(status.health.is_finite());
+        assert_ne!(status.sync_status, SyncStatus::Ok);
+        assert_ne!(status.sync_status, SyncStatus::Slow);
+    }
+
+    #[quickcheck]
+    fn test_sync_status_slow(duration: Duration) {
+        let duration = duration + Duration::from_secs(300);
+        let tipset = mock_tipset_at(
+            duration
+                .as_secs()
+                .saturating_sub(EPOCH_DURATION_SECONDS as u64 * 4),
+        );
+        let status = node_status(duration, tipset.as_ref());
+        assert!(status.health.is_finite());
+        assert_ne!(status.sync_status, SyncStatus::Behind);
+        assert_ne!(status.sync_status, SyncStatus::Ok);
+    }
+
+    #[test]
+    fn block_sync_timestamp() {
+        let duration = Duration::from_secs(60);
+        let tipset = mock_tipset_at(duration.as_secs() - 10);
+        let status = node_status(duration, tipset.as_ref());
+
+        assert!(status
+            .format(DateTime::<chrono::Utc>::MIN_UTC)
+            .contains("10s behind"));
+    }
+
+    #[test]
+    fn test_lag_uptime_ahead() {
+        let mut status = mock_node_status();
+        status.lag = -360;
+        assert!(status
+            .format(DateTime::<chrono::Utc>::MIN_UTC)
+            .contains("6m ahead"));
+    }
+
+    #[test]
+    fn chain_status_test() {
+        let duration = Duration::from_secs(100_000);
+        let tipset = mock_tipset_at(duration.as_secs() - 59);
+        let status = node_status(duration, tipset.as_ref());
+        let expected_status_fmt =
+            "[sync: Slow! (59s behind)] [basefee: 0 FIL] [epoch: 0]".to_string();
+        assert!(status
+            .format(DateTime::<chrono::Utc>::MIN_UTC)
+            .contains(&expected_status_fmt));
+
+        let tipset = mock_tipset_at(duration.as_secs() - 30000);
+        let status = node_status(duration, tipset.as_ref());
+
+        let expected_status_fmt =
+            "[sync: Behind! (8h 20m behind)] [basefee: 0 FIL] [epoch: 0]".to_string();
+        assert!(status
+            .format(DateTime::<chrono::Utc>::MIN_UTC)
+            .contains(&expected_status_fmt));
+    }
+
+    // FIXME: enable/debug this failing when color output is supported.
```

Comment:
I'd greatly prefer it if this uncommented code was removed before merging.

Reply:
removed the commented test